web-registry/src/pages/CreditClassDetails.tsx

```diff
@@ -115,215 +31,55 @@ function CreditClassDetails(): JSX.Element {
 }
 
 function CreditClassDetail({ isLandSteward }: CreditDetailsProps): JSX.Element {
-  const [modalIframeLink, setModalIframeLink] = useState<string>('');
-  const [isBuyerModalOpen, setBuyerModalOpen] = useState(false);
-  const [submitted, setSubmitted] = useState(false);
-  const styles = useStyles();
-  const navigate = useNavigate();
-
   let { creditClassId } = useParams();
+  const [ledgerClass, setLedgerClass] = useState<any>(undefined);
+  const [metadata, setMetadata] = useState<any>(undefined);
 
   const { data } = useAllCreditClassQuery({ client });
+  const { data: dbData } = useCreditClassByOnChainIdQuery({
+    variables: { onChainId: creditClassId?.toUpperCase() || '' },
```

Comment:
we should just have `credit_class/C01` for instance as path, no need to make it uppercase
URLs are case-sensitive

Reply:
URLs typically aren't uppercase though. This would cover the case of a link that is shared or typed lowercase. Maybe it's an edge case, but seems common enough to me.

Reply:
> URLs typically aren't uppercase though. 

https://schema.org/url and https://schema.org/URL are two different URLs for instance.

> And actually, it seems that react-router or the browser is not permitting an uppercase path - it resets it to lowercase.

there's a `sensitive` option for Route's in react-router: https://v5.reactrouter.com/core/api/Route/sensitive-bool
I think we should keep the case-sensitiveness as this might be confusing for users to know whether the class id is `C01` or `c01`.

Reply:
Sounds good. It is called `caseSensitive` now. Updated